src/sys/socket/consts.rs

```diff
@@ -1,75 +1,253 @@
 pub use self::os::*;
 
-#[cfg(any(target_os = "linux", target_os = "android"))]
 mod os {
     use libc::{self, c_int, uint8_t};
 
     pub const AF_UNIX: c_int  = libc::AF_UNIX;
```

Comment:
I'm wondering then if this `consts` module should not be removed completely in favor of libc ?